src/ActionModal/components/ActionModal.vue

```diff
@@ -451,11 +451,16 @@ export default {
         this.networkId.startsWith(`terra`) &&
         this.transactionData.type === transactionTypes.SEND
       ) {
-        return this.maxDecimals(
-          Number(this.gasEstimate) * Number(this.gasPrice) +
-            Number(this.amount) * terraTax,
-          6
-        ) // TODO get precision from API
+        return this.getDenom === this.network.stakingDenom
+          ? this.maxDecimals(
+              Number(this.gasEstimate) * Number(this.gasPrice) * 1.5,
```

Comment:
refactor 1.5 into a `const GAS_ADJUSTMENT`

Reply:
is our gasPrice also `0.015` (per uluna)?

Reply:
no, it is `0.001` for uluna (same than for uatom)

Reply:
Let's fix that?

Reply:
ah true. Forgot about that recommendation. Let's do this

Reply:
...wait actually it is much higher no?

He said

```
const GAS_PRICE = '0.015'
const fee = Math.ceil(gas_estimate * 1.5 * GAS_PRICE)
```

whereas now we have 1e-8... something is off here

Reply:
naaa, that's only if you get the gas estimation from the Cosmos SDK node. sorry for not seeing this earlier

Reply:
Ok, so then we are good no? We are using the gas price the said for uluna, `0.015`. This is what you were asking, right? 

(hard to keep track of these async conversations)